core/server/helpers/register.js

```diff
@@ -19,7 +19,10 @@ function asyncHelperWrapper(hbs, name, fn) {
         }).catch(function asyncHelperError(err) {
             var wrappedErr = err instanceof errors.GhostError ? err : new errors.IncorrectUsageError({
                     err: err,
```

Comment:
The original error is tracked here 🤔 

So Ignition keeps the stack+message of the original error. The stack get's combined and we both see, a custom error message and the original error message.

https://github.com/TryGhost/Ignition/blob/master/lib/errors/index.js#L48

Reply:
Sure, and that's great cos we can see what the original error was. The problem is that in this one place, if the error is not already a GhostError then we know it is something really odd and the logs are currently causing confusion. 

This change should make it explicitly clear what the original error was, as well as showing us any properties that are being lost e.g error codes from other software.

Reply:
Fwiw we can get rid of this when we stop needing to debug these problems. I also want to add more error handling to ghost_head and as we learn what different errors look like we can add more explicit handling for handlebars errors here.

Does that makes sense?

Reply:
Sure let's try to see if there is more context of the inherited error 👍